### ISSUE TITLE
Adjustable list sync task schedule interval

### ISSUE THREAD

**Issue by ricowalker** [Sonarr/Sonarr#5011]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Seems there's been a ([poor](https://github.com/Sonarr/Sonarr/issues/4325)) attempt to request this feature in the past. 

My issue is with the import list feature. As of now I don't know of another way to have separate Sonarr instances and keeping these in sync for keeping different libraries.
My current connection allows me to use my Jellyfin instance to watch media on the go and locally. However my speeds and hardware aren't good for transcoding 4K content. For this reason I keep a separate 4K library on a separate instance of Sonarr for local use.

When adding Media to sonarr on the go I use the nzb360 app. The way I set it up a quality profile in the main sonarr instance which the 4K instance imports. This way I can have both a 1080p version and 4K version available.

### Describe the solution you'd like

In order to solve this i'd need an option for multiple quality options within a single instance, the ability to sync multiple instances, or my suggestion, the ability to change the task interval in order to keep the instance more synchronized as there is a big delay in between the time needed to gather the 4k version for it.

A simple drop-down menu in the task menu or an option within the Import list setting would help tons.

### Describe alternatives you've considered

Any other alternatives besides my current set-up seem cumbersome and unfriendly to use.
I've tried renaming the series 

### Anything else?

Used the following guide to attempt synced sonarr instances: [TraSH-Guides](https://trash-guides.info/Sonarr/Tips/Sync-2-radarr-sonarr/)
Having the same thing with Radarr

**Comment by riconnite** [comment]
> TL;DR - Add a List Import configurable time between say 6-24hrs like Radarr has?

Indeed, would give a little more flexibility.
